src/rexsdesktop/controller/User.java

```diff
@@ -582,8 +582,8 @@ public int getCantidadUsuarios() {
         try {
             db.NumUsuarios();
             int Numero;
-            System.out.println(CurrentUser.idTipoUsuario);
-            if (CurrentUser.idTipoUsuario == 1) {
+            System.out.println(CurrentUser.getIdTipoUsuario());
+            if (CurrentUser.getIdTipoUsuario() == 1) {
                 Numero = db.getCantidadUsuarios2();

```

Comment:
when you have a method with this kind of names `getCantidadesUsuarios2`,  means you need to think about if you are repeating code or need to clean up more your code